public/lib/helpers/timeline_helper.js

```diff
@@ -27,8 +27,10 @@ define(function (require) {
       let field;
       if (params.labelFieldSequence) { // in kibi, we have the path property of a field
         field = kibiUtils.getValuesAtPath(hit._source, params.labelFieldSequence);
-      } else {
-        field = _.get(hit._source, params.labelField);
+      } else { // create kibi path property from ES path by splitting on .
+        const kibanaSplit = params.labelField.split('.');
+        const kibanaSequence = Array.isArray(kibanaSplit) ? kibanaSplit : Array.of(kibanaSplit);
```

Comment:
The variable `kibanaSplit` is always an array, isn't it ? If so, why is the check `Array.isArray` needed ?